README.md

```diff
@@ -193,6 +193,7 @@ Questions can be asked by raising an `Issue`.
 - [Josh Callahan](https://github.com/joshcallahan)
 - [Joykishan Sharma](https://github.com/JoykishanSharma)
 - [Kabir Jain](https://github.com/kkkkkabir)
+- [kuwar kapur](https://github.com/kuwarkapur)
```

Comment:
@kuwarkapur add your name in alphabetical order, after **Krishna R**  `Line no. 199`